### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by molly-moen**
# Type of change? 

- [ ] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
Update readme to be in line with template and dotnet version

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [x] I have updated the documentation accordingly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,88 +3,116 @@
...
+project to provide a generic implementation for an end-to-end IoT solution.
+More information [here][rm-arch-url].
 
-### 1. Environment Variables
+## Features
+* Create or update device groups
+* Get all or a single device group
+* Get or upload logo
+* Set or get overall solution settings
+* Set or get individual user settings
```
Comment (ppathan) [inline README.md]: nit: `Get or set`

File: README.md [hunk 2]
```diff
@@ -3,88 +3,116 @@
...
-docker-compose up
-```
-
-The Docker compose configuration requires the `PCS_STORAGEADAPTER_WEBSERVICE_URL` environment variable.
-
-Running the service in an IDE
-=============================
 ## Prerequisites
+- Install IntelliJ IDEA: https://www.jetbrains.com/idea/download
 - Install SBT: http://www.scala-sbt.org/download.html
```
Comment (ppathan) [inline README.md]: Add links through []() pattern

File: README.md [hunk 3]
```diff
@@ -3,88 +3,116 @@
...
-- Install the latest Java SDK:
-  http://www.oracle.com/technetwork/java/javase/downloads/index.html
-
-## Run and Debug with IntelliJ
-
-Install Intellij IDEA Community: https://www.jetbrains.com/idea/download
+- Install the latest Java 8 SDK: 
+  http://www.oracle.com/technetwork/java/javase/downloads
```
Comment (ppathan) [inline README.md]: nit: same comment about links

File: README.md [hunk 4]
```diff
@@ -3,88 +3,116 @@
...
-
-* "Open" the project with IntelliJ, the IDE should automatically recognize the
-  SBT structure. Wait for the IDE to download some dependencies (see IntelliJ
-  status bar)
-* Create a new Run Configuration, of type "SBT Task" and enter "run 9004"
-  (including the double quotes). This ensures that the service to start using
-  the TCP port 9004
+Steps using IntelliJ IDEA Community, with SBT plugin enabled:
+
+* Make sure the [prerequisites](#prerequisites) are set up.
```
Comment (ppathan) [inline README.md]: In other repos we use `P` capital. I actually like lower case better, its up to you

Comment (molly-moen) [reply README.md]: I also like lower case better

**Pull request merged**